### PR TITLE
refactor ucall to accommodate cc requirement

### PR DESCRIPTION
**Pull request by yaoxin1995**

### CODE REVIEW

**Comment by shrik3** [comment]
Why would this series have a [update tool chain to 2023-12-11](https://github.com/QuarkContainer/Quark/pull/1186/commits/e653f0ebb590f41830ba92eaa7b5a2ac62d3ee3c) commit? isn't already merged into main? Perhaps rebase against main first?

File: config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "DebugLevel"    : "Error",
```
Comment (shrik3) [inline config.json:2]: do not modify config here

File: qkernel/makefile [hunk 2]
```diff
@@ -43,16 +43,16 @@ $(cc_kernel_debug): cc_kernel_debug $(assembly_object_files)
...
 kernel_debug:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json
+	CARGO_TARGET_DIR=../target cargo +nightly-2023-12-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json
 
 cc_kernel:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --release --features cc
+	CARGO_TARGET_DIR=../target cargo +nightly-2023-12-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --release --features cc
 
 cc_kernel_debug:
-	CARGO_TARGET_DIR=../target cargo +nightly-2022-08-11-x86_64-unknown-linux-gnu xbuild --target $(arch)-qkernel.json --features cc
```
Comment (shrik3) [inline qkernel/makefile:55]: please drop this, as it's already merged into main. (It might bite you XD)

Comment (yaoxin1995) [reply qkernel/makefile:55]: Thank you, will do it.

**Pull request merged**